src/App/Layers/GameLayer.cpp

```
@@ -424,9 +424,15 @@ void GameLayer::OnUpdate(float dt)
     if (transform && cam)
     {
         float speed = 2.0f;
+        float rotSpeed = 1.f;
         float zoomSpeed = 100.f;
 
         glm::vec3 input(0);
+        glm::vec2 rot(0.f);
+        if (InputSystem::IsKeyHeld(GLFW_KEY_LEFT_SHIFT))
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)